### DISCUSSION TITLE
Help on to convert ImageFrame to BufferedImage for depthai

### DISCUSSION THREAD

**Discussion by mksaaaaaaaaa** [bytedeco/javacpp-presets#1485] [General]
Hi Team,
can anyone help me to convert below objects.
1) org.bytedeco.opencv.opencv_core.Mat into java.awt.image.BufferedImage
2) org.bytedeco.depthai.ImgFrame into java.awt.image.BufferedImage.

thanks.

**Comment by saudet** [comment]
Please use Java2DFrameConverter and OpenCVFrameConverter:
http://bytedeco.org/news/2015/04/04/javacv-frame-converters/

**Reply by saudet** [reply]
However, there isn't a DepthAIFrameConverter, yet, but contributions are welcome!
